### ISSUE TITLE
Enhancement: Make clear which account is current account in the account-list (i.e. Balance for different Alby accounts not shown in dropdown)

### ISSUE THREAD

**Issue by pavanjoshi914** [getAlby/lightning-browser-extension#966]
### Is your feature request related to a problem? Please describe

Currently dropdown doesn't show the balance with the account associated. this leads to the confusion that on which account currently user is. or if the user wants to pay by seeing which account has how much balance the only way is to switch to that account.

### Describe the solution you'd like

Show account name with balance in the dropdown as well

### Additional context

![image](https://user-images.githubusercontent.com/55848322/170821077-488cd05b-f495-4fe4-8143-398b2250aedc.png)

**Comment by thebrandonlucas** [comment]
To add to this, I think it may also be helpful to have the Accounts screen also show the id/value of the account, rather than just the initial name, since adding two different accounts of the same type would lead to the same name. If you're deleting, It's difficult to know which one is which in that scenario as well
<img width="1056" alt="Screen Shot 2022-05-29 at 6 27 32 PM" src="https://user-images.githubusercontent.com/38222767/170893778-959d9f94-1b88-4fc6-a15e-04880c6c75a5.png">

**Comment by escapedcat** [comment]
> I think it may also be helpful to have the Accounts screen also show the id/value of the account

@thebrandonlucas this might be easier to be handled in a separate issue. Wanna open one?

**Comment by bumi** [comment]
Currently we can only do requests (like getting the balance) for the selected account. This is because some accounts need to first "boot up" and initialize the connection. 

In the latest version a checkmark is shown for the currently selected account. Do you think that is enough?

**Comment by escapedcat** [comment]
> In the latest version a checkmark is shown for the currently selected account. Do you think that is enough?

Good point. The main point of this issue was, that it's unclear which is the currently selected account in the list (updated the issue title). This has been done.

Showing all balances would be a nice to have but it's not crucial.

**Comment by theewiz** [comment]
Hi everyone, do check this out. On this sample Account design, I have added a checkmark to differentiate accounts that are currently in use. Also, the balances and unique IDs of other accounts has been added to the accounts listed below

Do share your thoughts, if there is any additions/corrections. Thanks!


![wwww](https://user-images.githubusercontent.com/81051645/173245446-2aaadbb9-3088-43b9-95de-d0a1b26f634e.png)
